nodes/liveplot

```diff
@@ -0,0 +1,211 @@
+        for objId in objIds_in_data:
+            if objId not in self.lines:
+                # add new line
+                self.lines[objId], = self.ax.plot(da[objId]['x'],
+                                                  da[objId]['y'],
+                                                  da[objId]['z'])
+                self.pts[objId], = self.ax.plot([da[objId]['x'][-1]],
+                                                [da[objId]['y'][-1]],
+                                                [da[objId]['z'][-1]],
+                                                'o',
+                                                c="r")
+            # load data from mem to lines
+            self.lines[objId].set_data(da[objId]['x'], da[objId]['y'])
+            self.lines[objId].set_3d_properties(da[objId]['z'])  # this line is needet to draw in this plot
+            self.pts[objId].set_data([da[objId]['x'][-1]], [da[objId]['y'][-1]])
+            self.pts[objId].set_3d_properties([da[objId]['z'][-1]])  # this line is needet to draw in this plot
+            # autoscale
+            if self.autoscale:
+                min_max = self.data.get_max_min()
+                if self.autoscale == 'aspect_on':
+                    vals = [math.sqrt(min_max[i] ** 2) for i in min_max]
+                    limit = max(vals)
+                    self.ax.set_xlim3d(-limit, limit)
+                    self.ax.set_ylim3d(-limit, limit)
+                    self.ax.set_zlim3d(-limit, limit)
+                else:
+                    feix_me = 0.000000000001  # FIXME: else me set min max val the smae, amd matplotlib does not like this
+                    self.ax.set_xlim3d(min_max['min_x'], min_max['max_x'] + feix_me)
+                    self.ax.set_ylim3d(min_max['min_y'], min_max['max_y'] + feix_me)
+                    self.ax.set_zlim3d(min_max['min_z'], min_max['max_z'] + feix_me)
+        self._delete_dead_lines(objIds_in_data)
+        return self.lines, self.pts
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description='Alternative to rviz, listens to flydra_mainbrain_super_packets')
+    parser.add_argument('--x_min',  dest='x_min', type=int, default=-0.2, help='drawing limits')
+    parser.add_argument('--x_max', dest='x_max', type=int, default=0.2, help='drawing limits')
+    parser.add_argument('--y_min', dest='y_min', type=int, default=-0.2, help='drawing limits')
+    parser.add_argument('--y_max', dest='y_max', type=int,  default=0.2, help='drawing limits')
+    parser.add_argument('--z_min', dest='z_min', type=int, default=-0.2, help='drawing limits')
+    parser.add_argument('--z_max', dest='z_max', type=int, default=0.2, help='drawing limits')
+    parser.add_argument('--autoscale', dest='autoscale', default=False, choices=('aspect_on', 'aspect_off'), help='set drawing limits automaticly')
+    parser.add_argument('--update_rate', dest='update_rate', type=int, default=10, help='in Hz, should not be higher than 20Hz')
```

Comment:
if it shouldn't be higher than 20, just take the user choice and call min(args.update_rate,20)